Broken hyperlink returns 404 (create linux vhd)

kaushalp opened the pull request:
The hyper link for creating and uploading vhd is broken and points to "classic/create-upload-vhd.md?toc=%2fazure%2fvirtual-machines%2flinux%2fclassic%2ftoc.json"

I have updated the doc to point to https://docs.microsoft.com/en-us/azure/virtual-machines/linux/create-upload-generic

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

kaushalp committed:
Broken hyperlink returns 404 (create linux vhd)

The hyper link for creating and uploading vhd is broken and points to "classic/create-upload-vhd.md?toc=%2fazure%2fvirtual-machines%2flinux%2fclassic%2ftoc.json"

I have updated the doc to point to https://docs.microsoft.com/en-us/azure/virtual-machines/linux/create-upload-generic

GitHubber17 committed:
Update endorsed-distros.md
